blivet/devices/disk.py

```diff
@@ -95,6 +96,23 @@ def __init__(self, name, fmt=None,
                                serial=serial, model=model,
                                vendor=vendor, bus=bus)
 
+        try:
+            ssd = int(util.get_sysfs_attr(self.sysfs_path, "queue/rotational")) == 0
+        except TypeError:  # get_sysfs_attr returns None from all error paths
+            ssd = False
+
+        self.tags.add(Tags.local)
+        if ssd:
+            self.tags.add(Tags.ssd)
+        if bus == "usb":
+            self.tags.add(Tags.usb)
+        if self.removable:
+            self.tags.add(Tags.removable)
+
+    def _clear_local_tags(self):
+        local_tags = set([Tags.local, Tags.ssd, Tags.usb, Tags.removable])
```

Comment:
This can be written as ``{Tags.local, Tags.ssd, Tags.usb, Tags.removable}`` which avoids creation of the temporary list. But I know it looks quite confusing due to the same notation for dicts.